stdlib/universe/covariance_test.go

```diff
@@ -88,11 +88,7 @@ func TestCovariance_NewQuery(t *testing.T) {
 					{
 						ID: "join2",
 						Spec: &universe.JoinOpSpec{
-							On: []string{"host"},
-							TableNames: map[flux.OperationID]string{
-								"from0": "x",
-								"from1": "y",
-							},
```

Comment:
A handful of tests "want" the `TableNames` field removed in this diff. It's a bit of a shame the `params` field is not public, but _there are plenty of acceptance tests_ already in place to attest to the name-to-tables handling.

After looking at it, I don't feel like it's important to rush to rework things to get visibility on this at the OpSpec level.